packages/flutter/lib/src/painting/text_style.dart

```diff
@@ -1324,18 +1322,17 @@ class TextStyle with Diagnosticable {
       locale,
       foreground,
       background,
-      hashList(shadows),
-      hashList(fontFeatures),
+      Object.hashAll(shadows ?? <Object>[]),
+      Object.hashAll(fontFeatures ?? <Object>[]),
       decoration,
       decorationColor,
       decorationStyle,
       decorationThickness,
       fontFamily,
-      hashList(fontFamilyFallback),
+      Object.hashAll(fontFamilyFallback ?? <Object>[]),
       _package,
       overflow,
     ]);
-  }
```

Comment:
as above (three const, indentation)

Reply:
Do I need to add `?? const <Object?>[]` if the variable is non-nullable?

Reply:
oh, no, ignore that :-)